docs/UserGuide.md

```diff
@@ -71,12 +71,19 @@ Format: `help`
 
 ### Adding a food item: `add`
 
-*placeholder*
+Add food items into your fridge.
 
-Format: `placeholder`
+Format: `add`
 
 Examples:
-* `placeholder`
+* `add Spinach`
+* `add Meiji Milk`
+* `add Awfully Chocolate`
+
+Command outcome:
```

Comment:
the format of the example differs from the current format that is currently in the user guide now, may need to change the format